modules/@apostrophecms/schema/ui/apos/logic/AposSchema.js

```diff
@@ -1,4 +1,3 @@
-import { detectFieldChange } from 'Modules/@apostrophecms/schema/lib/detectChange';
```

Comment:
Maybe we should add a comment saying this method is not used anymore or remove it? I don't know if we mentioned it in the docs.